index.html

```diff
@@ -75,7 +75,7 @@ <h4>Setup</h4>
             </tr>
 
             <tr>
-                <td><label for="floppy_image">Floppy disk image</label></td>
+                <td><label for="floppy_image">Floppy image / bzImage / vmlinuz.bin</label></td>
```

Comment:
I think it would be better to add a separate input for this.

Reply:
Yes. I just wanted less buttons on screen for user.